_posts/en/meetings/2016-06-30-meeting.md

```diff
@@ -0,0 +1,121 @@
+
+The release notes need a bunch of writeups for all the mining changes.
+
+### meeting conclusion
+
+- Remove -blockminsize and give warning when the argument is given
+- Update release notes with the mining changes
+
+## Segwit
+ 
+### background
+ 
+Developers are working on a soft fork to introduce segregated witness onto Bitcoin mainnet. Segregated witness (segwit) allows transaction signature data to be stored outside of the data hashed to produce transaction identifiers, removing all known forms of third-party malleability, allowing full nodes to compile the current UTXO set without downloading all signatures, and laying the groundwork for fraud proofs that can allow lightweight (SPV) clients to help enforce more of the consensus rules. The segwit soft fork also allows miners to substitute 1 byte of block space with 4 bytes of segwit data, increasing transaction capacity for wallets that use segwit. segregated witness BIPs: [BIP141][], [BIP142][], [BIP143][], [BIP144][] and [BIP145][]
+
+### meeting comments
+
+During his [review](https://petertodd.org/2016/segwit-consensus-critical-code-review) petertodd found a mempool DoS risk due to malleated transactions ([issue 8279](https://github.com/bitcoin/bitcoin/issues/8279)).
+
+Gmaxwell proposes to make low-dos-score a ranking criteria in the eviction protection logic, which would make running with a very high DoS threshold more reasonable. Sipa notes the DoS score should also attenuate over time if such changes are made, otherwise longer-lived connections will accumulate a score they don't deserve.
+
+Petertodd would like to see different thresholds periods for some nodes, so that while you won't waste bandwidth on everyone a few peers are connected regardless. Gmaxwell notes such a thing could turn those peers into blockonly as that's all we care about for anti-partitioning and it almost completely eliminates DoS concerns.
+
+Sipa thinks there may be reasons to introduce something like a "resource usage score" which is distinct from "misbehavior", which gets used to decide which peers to disconnect in favor of others, but never causes a ban. 
+
+Gmaxwell notes bitcoinXT recently started making only outbound connections to other XT nodes, which in combination with segwit will partition them. An issue has been made in the XT repo for it.
+
+### meeting conclusion
+
+- Brainstorm about connection management stuff
+
+## Dbcache
+
+### background
+ 
+Gmaxwell noticed very slow reindexing while testing with default dbcache, which got confirmed by sipa who saw similar behavior. This was brought up in [last weeks meeting](https://bitcoincore.org/en/meetings/2016/06/23/#perceived-validation-slowdowns).
```

Comment:
links can be relative without the domain prefix. /en/meetings/2016/06/23/#perceived-validation-slowdowns